nixos/modules/config/no-x-libs.nix

```diff
@@ -30,8 +30,8 @@ with lib;
       beam = super.beam_nox;
       cairo = super.cairo.override { x11Support = false; };
       dbus = super.dbus.override { x11Support = false; };
-      ffmpeg_4 = super.ffmpeg_4.override { ffmpegVariant = "headless"; };
-      ffmpeg_5 = super.ffmpeg_5.override { ffmpegVariant = "headless"; };
+      ffmpeg_4 = super.ffmpeg_4-headless;
```

Comment:
please revert https://github.com/NixOS/nixpkgs/commit/9e3e928ad45fa9df11280be4ff581c4c11992e21

Reply:
Removed the commit